compose/material/material/src/desktopMain/kotlin/androidx/compose/material/DesktopMenu.desktop.kt

```diff
@@ -115,12 +123,31 @@ fun DropdownMenu(
                 }
             },
         ) {
+            val focusManager = LocalFocusManager.current
+
             DropdownMenuContent(
                 expandedStates = expandedStates,
                 transformOriginState = transformOriginState,
-                modifier = modifier,
+                modifier = modifier.onKeyEvent {
+                    if (it.type != KeyEventType.KeyDown) return@onKeyEvent false
+                    when (it.key) {
+                        Key.DirectionDown -> {
```

Comment:
I realised, that after we merge the other PR, we should switch `inputMode` somehow when we move the focus with arrow keys

Reply:
seems to be not a problem:

```
val inputModeManager = LocalInputModeManager.current
.... 
inputModeManager.requestInputMode(InputMode.Keyboard)
```

Reply:
That should work )